src/liveshare/share.ts

```diff
@@ -0,0 +1,336 @@
+import * as vscode from 'vscode';
```

Comment:
A design pattern I personally like (e.g. in the helpViewer files) is to name the main file `index.ts` and only import from this file elsewhere in the extension. If necessary, items can also be re-exported here.

This makes it easier to work with/on the liveshare related functions, since you have a single file from where you import these, and you only have to check the impact on the other liveshare files when changing liveshare-related functions that are not exported in `index.ts`.

I think that's just personal preference though, so feel free to mark as resolved if you don't want to change this!

Reply:
Ah, that's pretty elegant! Thank you for the tip.